katas/main/src/pokerhands/valeria/Dealer.cs

```diff
@@ -0,0 +1,49 @@
+﻿namespace PokerHands.Valeria
+{
+    public class Dealer
+    {
+        public Player WhitePlayer, BlackPlayer;
+        public int BlackPlayerPoints = 0, WhitePlayerPoints = 0;
+        private readonly Dictionary<string, int> PriorityRulesOrderList = new Dictionary<string, int> { { "with straight flush", 9 }, { "with four of a kind", 8 }, { "with full house", 7 }, { "with flush", 6 }, { "with straight", 5 }, { "with three of a kind", 4 }, { "with two pairs", 3 }, { "with pair", 2 }, { "with high card", 1 } };
```

Comment:
This line is too long, consider wrapping it in multiple lines so it is easier to read